src/test/java/org/folio/rest/impl/MockServer.java

```diff
@@ -104,6 +106,7 @@ private Router defineRoutes() {
     router.route(HttpMethod.GET, resourcesPath(INVOICES)).handler(this::handleGetInvoices);
     router.route(HttpMethod.GET, resourceByIdPath(INVOICES, ID_PATH_PARAM)).handler(this::handleGetInvoiceById);
     router.route(HttpMethod.GET, resourcesPath(FOLIO_INVOICE_NUMBER)).handler(this::handleGetFolioInvoiceNumber);
+    router.route(HttpMethod.GET, resourcesPath(INVOICE_LINES)+"/:id").handler(this::handleGetInvoiceLinesById);
```

Comment:
1. Please add space before and after `+` sign or use `resourceByIdPath` method
2. Use `ID_PATH_PARAM` constant